packages/react-components/react-combobox/package.json

```diff
@@ -41,6 +41,7 @@
     "@fluentui/react-shared-contexts": "^9.3.3",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

Comment:
_nit:_ Can you move this above `@fluentui/react-portal` in line `39` to keep alphabetical order?